src/APCuL1.php

```diff
@@ -38,9 +26,10 @@ public function getKeyOverhead(Address $address)
         // decrementing the overhead by existing hits when an item is set. This
         // would make hits cheaper but writes more expensive.
 
+        $success = null;
         $apcu_key = $this->getLocalKey($address);
         $overhead = apcu_fetch($apcu_key . ':overhead', $success);
-        if ($success) {
+        if ($success === true) {
```

Comment:
Can we swap these to be `true === $success` here and in similar places. It just minimizes the chance of accidentally assigning instead of comparing. I'll probably fix all other conditions at some point to be this way.